src/u/config.php

```diff
@@ -3,5 +3,8 @@
 $settings = file_get_contents(__DIR__ . "/setup/settings.json");
 $settings = json_decode($settings, JSON_UNESCAPED_SLASHES);
 
+$version = ['version' => '1.0.0'];
```

Comment:
I'm not sure we should introduce version strings, I'd like to keep this project relatively simple :)

Reply:
removed in the latest PR